whelktool/scripts/cleanups/2020/12/lxl-2612-byggnadsstyrelsen-broken-links.groovy

```diff
@@ -0,0 +1,62 @@
+import whelk.util.DocumentUtil
+
+String where = """
+    collection = 'bib' 
+    AND (data#>>'{@graph,1,associatedMedia}' LIKE '%http://www.fastighetswiki.se/wiki%' 
+        OR data#>>'{@graph,1,marc:versionOfResource}' LIKE '%http://www.fastighetswiki.se/wiki%'
+        OR data#>>'{@graph,1,isPrimaryTopicOf}' LIKE '%http://www.fastighetswiki.se/wiki%'
+        OR data#>>'{@graph,1,relatedTo}' LIKE '%http://www.fastighetswiki.se/wiki%')
+    """
+
+List keys856 = ['associatedMedia', 'marc:versionOfResource', 'isPrimaryTopicOf', 'relatedTo']
+
+List controlNumbers= []
+List ids = []
+
+// First get id and controlNumber of the fastighetswiki records
+selectBySqlWhere(where) { data ->
+    controlNumbers << data.doc.getControlNumber()
+    ids << data.doc.getShortId()
+}
```

Comment:
Not thread-safe, unfortunately.

Can do for example:
```
List controlNumbers = Collections.synchronizedList([])
List ids = Collections.synchronizedList([])
```

Reply:
Right...fixed!